std/portable/index.js

```diff
@@ -206,10 +206,18 @@ globalScope["isFloat"] = function isFloat(arg) {
   return typeof arg === "number";
 };
 
+globalScope["isNullable"] = function isNullable(arg) {
+  return arg == null;
```

Comment:
That's a bit thin, not sure polyfilling this makes sense.

Reply:
Why not? Just for compatibility. AS has this built-in

Reply:
Yeah, but AS checks whether a type is actually nullable, while in JS it doesn't work with anything but actual `null`, leading to entirely different semantics.

Reply:
Looking at it the other way around, in JS everthing is potentially nullable, so this function should either always return `true` or adhere to what could, potentially, be nullable in a portable scenario, which is any reference but not `number` or `boolean`.

Reply:
Ahh, got it. I will return just `true`